tutorials/dapt-curation/code/utils.py

```diff
@@ -282,6 +296,79 @@ def dedupe(dataset: DocumentDataset) -> DocumentDataset:
     return DocumentDataset(deduped)
 
 
+def fuzzy_dedupe(dataset: DocumentDataset, type: str = "text") -> DocumentDataset:
+    """
+    Removes near-duplicate documents and code lines
+
+    Args:
+        dataset (DocumentDataset): The dataset containing documents.
+        type (str): Document type to process.
+
+    Returns:
+        DocumentDataset: The deduplicated dataset.
+    """
+    cache_dir = f"./workspace/fuzzy_dedupe_cache/{type}"
```

Comment:
Instead of hardcoding, can you pass it into the function instead? Meaning

```
def fuzzy_dedupe(dataset: DocumentDataset, type: str = "text", cache_dir: str) -> DocumentDataset:
```

Then from inside the main curation loop, we can pass in a value that is relative to the script path, like:

```
CACHE_DIR = os.path.join(SCRIPT_DIR_PATH, "cache")
fuzzy_dedupe(..., cache=CACHE_DIR)

```